src/execution/vm/bytecode_generator.cpp

```diff
@@ -2375,7 +2395,15 @@ void BytecodeGenerator::VisitBuiltinCallExpr(ast::CallExpr *call) {
     case ast::Builtin::Cos:
     case ast::Builtin::Cot:
     case ast::Builtin::Sin:
-    case ast::Builtin::Tan: {
+    case ast::Builtin::Floor:
+    case ast::Builtin::Ceil:
+    case ast::Builtin::Log10:
+    case ast::Builtin::Tan:
+    case ast::Builtin::Truncate: {
+      VisitBuiltinTrigCall(call, builtin);
+      break;
+    }
```

Comment:
Why have this when the next case (`ast::Builtin::Log2`) also calls `VisitBuiltinTrigCall`?

Reply:
Fixed